schema.sql

```diff
@@ -0,0 +1,33 @@
+DROP DATABASE IF EXISTS ads;
+
+CREATE DATABASE ads;
+
+USE ads;
+
+CREATE TABLE similar_listing (
+  objectID INTEGER AUTO_INCREMENT PRIMARY KEY,
+  photoID INTEGER,
+  listingtype CHAR(100),
+  caption CHAR(200),
+  price INTEGER,
+  rating INTEGER
+);
+
+CREATE TABLE things_to_do (
+  objectID INTEGER AUTO_INCREMENT PRIMARY KEY,
+  photoID INTEGER,
+  eventtype CHAR(100),
+  caption CHAR(200),
+  price INTEGER,
+  rating INTEGER
+);
+
+		
+CREATE TABLE rooms (
+  id INTEGER AUTO_INCREMENT PRIMARY KEY,
+  roomname CHAR(50)
+);
+
+
+ALTER TABLE listing ADD FOREIGN KEY (offering) REFERENCES offering (id);
```

Comment:
Where are your "listing" and "event" table referenced?

Reply:
Thanks for the comments. I have updated the table. There was some issue with the git branching.

Reply:
Please review.

Reply:
How are you tying the `similar_listings` and `things_to_do` with a given offering id? I would expect to see an `offerings` table along with an `offerings_id` as a foreign key in both of your tables.